exchanges/gateio/gateio_types.go

```diff
@@ -2112,12 +2112,15 @@ type WsUserPersonalTrade struct {
 
 // WsSpotBalance represents a spot balance.
 type WsSpotBalance struct {
-	Timestamp types.Time   `json:"timestamp_ms"`
-	User      string       `json:"user"`
-	Currency  string       `json:"currency"`
-	Change    types.Number `json:"change"`
-	Total     types.Number `json:"total"`
-	Available types.Number `json:"available"`
+	Timestamp    types.Time    `json:"timestamp_ms"`
+	User         string        `json:"user"`
+	Currency     currency.Code `json:"currency"`
+	Change       types.Number  `json:"change"`
+	Total        types.Number  `json:"total"`
+	Available    types.Number  `json:"available"`
+	Freeze       types.Number  `json:"freeze"`
+	FreezeChange types.Number  `json:"freeze_change"`
+	ChangeType   string        `json:"change_type"` // Type of change, e.g. "order-create", "order-match"
```

Comment:
🦍 Very optional nitpick
```suggestion
	ChangeType   string        `json:"change_type"` // e.g. "order-create", "order-match"
```